src/apps/frontend/pages/authentication/login/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import AuthenticationFormLayout from '../authentication-form-layout';
+import AuthenticationPageLayout from '../authentication-page-layout';
+
+import LoginForm from './login-form';
+
+export const Login: React.FC = () => (
+  <AuthenticationPageLayout>
+    <AuthenticationFormLayout>
+      <LoginForm></LoginForm>
```

Comment:
`onSuccess`, `onError`.

The page should be responsible to show error if its an error via toast, or navigate to `/` (or show toast for now) if success

Reply:
`<LoginForm onSuccess ={onSuccess} onError ={onError}></LoginForm>`